### PR TITLE
Fix running single test with `task`

### PR DESCRIPTION
**Pull request by w84thesun**
# Description

## Readiness checklist

- [ ] I added/updated unit tests.
- [ ] I added/updated integration/compatibility tests.
- [ ] I added/updated comments and checked rendering.
- [ ] I made spot refactorings.
- [x] I updated user documentation.
- [ ] I ran `task all`, and it passed.
- [x] I ensured that PR title is good enough for the changelog.
- [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Labels, Project and project's Sprint fields.
- [x] I marked all done items in this checklist.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -174,7 +174,7 @@ For example:
 
 - to run a single test case for in-process FerretDB with `pg` handler
   with all subtests running sequentially,
-  you may use `env GOFLAGS='-run=TestName/TestCaseName -parallel=1' task test-integration-pg`;
+  you may use `env RUN='TestName/TestCaseName' task test-integration-pg`;
```
Comment (AlekSi) [inline CONTRIBUTING.md:177]: The text above should be updated: it mentions only `GOFLAGS`, says "with all subtests running sequentially" (that is not true in that example), etc

File: Taskfile.yml [hunk 2]
```diff
@@ -189,72 +189,72 @@ tasks:
     dir: integration
     cmds:
       - >
-        go test -count=1 -run='{{.RUN}}' -timeout={{.INTEGRATIONTIME}} {{.RACEFLAG}} -tags={{.BUILDTAGS}} -shuffle=on -coverpkg=../...
+        go test -count=1 -run='{{or .RUN .SHARD_RUN}}' -timeout={{.INTEGRATIONTIME}} {{.RACEFLAG}} -tags={{.BUILDTAGS}} -shuffle=on -coverpkg=../...
```
[the remark this answers is not in the document]
Comment (AlekSi) [reply Taskfile.yml:192]: Also, `TEST_RUN` should be defined on top of the file

[the remark this answers is not in the document]
Comment (w84thesun) [reply Taskfile.yml]: But it's intended to be passed with the environment variables. It won't work if I define it in the predefined variables list.

[the remark this answers is not in the document]
Comment (AlekSi) [reply Taskfile.yml]: `RUN` / `TEST_RUN` is not an environment variable; that's Task's variable. The ones on top of the file are "public" variables; in contrast, `SHARD_RUN` is a "private" variable

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -169,19 +169,19 @@ You can also run `task -C 1` to limit the number of concurrent tasks, which is u
...
 you may also use all standard `go` tool facilities,
-including [`GOFLAGS` environment variable](https://pkg.go.dev/cmd/go#hdr-Environment_variables).
+including [`GOFLAGS` environment variable](https://pkg.go.dev/cmd/go#hdr-Environment_variables)
+or predefined variable `TEST_RUN`.
 For example:
 
 - to run a single test case for in-process FerretDB with `pg` handler
-  with all subtests running sequentially,
-  you may use `env GOFLAGS='-run=TestName/TestCaseName -parallel=1' task test-integration-pg`;
+  you may use `env TEST_RUN='TestName/TestCaseName' task test-integration-pg`;
```
Comment (AlekSi) [inline CONTRIBUTING.md:177]:
```suggestion
  you may use `env GOFLAGS='-parallel=1' task test-integration-pg TEST_RUN='TestName/TestCaseName' `;
```

And please add a text about sequential tests back

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -169,12 +169,13 @@ You can also run `task -C 1` to limit the number of concurrent tasks, which is u
 
 Finally, since all tests just run `go test` with various arguments and flags under the hood,
 you may also use all standard `go` tool facilities,
-including [`GOFLAGS` environment variable](https://pkg.go.dev/cmd/go#hdr-Environment_variables).
+including [`GOFLAGS` environment variable](https://pkg.go.dev/cmd/go#hdr-Environment_variables)
+or predefined variable `TEST_RUN`.
```
Comment (AlekSi) [inline CONTRIBUTING.md:173]: The resulting text is simply not correct: `TEST_RUN` is not a "standard `go` tool facility"

**Review by noisersup — APPROVED** [review]
LGTM

**Review by AlekSi — APPROVED** [review]

**Pull request merged**